File: PontoFacil.Api/Controlador/ExposicaoDeEndpoints/v1/DTO/DoServidorParaCliente/UsuarioPesquisadoDTO.cs
namespace PontoFacil.Api.Controlador.ExposicaoDeEndpoints.v1.DTO.DoServidorParaCliente;
public class UsuarioPesquisadoDTO
{
    public int Id { get; set; }
    public string? Nome { get; set; }
    public string? CPF { get; set; }
    public DateTime DataNascimento { get; set; }
}

File: ponto-facil/src/chamada-api/DTO/DoServidorParaCliente/UsuarioPesquisadoDTO.js
export default class UsuarioPesquisadoDTO {
  get id() {
    return this._id;
  }
  set id(value) {
    this._id = value;
  }
  get nome() {
    return this._nome;
  }
  set nome(value) {
    this._nome = value;
  }
  get cpf() {
    return this._cpf;
  }
  set cpf(value) {
    this._cpf = value;
  }
  get dataNascimento() {
    return this._dataNascimento;
  }
  set dataNascimento(value) {
    this._dataNascimento = value;
  }
}
